CIPs/cip-0051.md

```diff
@@ -0,0 +1,155 @@
+---
+cip: 51
+title: Federated Attestations Protocol
+author: @eelanagaraj, @isabellewei
+discussions-to: <URL>
+status: Draft
+type: Standards
+category (*only required for Standards Track): Ring 0
+created: 2022-06-29
+license: Apache 2.0
+---
+
+## Overview
+
+This CIP proposes a new version (from here referred to as ASv2) of the existing validator-run attestation protocol (CIP01, from here referred to as ASv1), which will leverage a federated set of independent issuers to perform attestations. It will also extend attestations beyond phone numbers to other types of off-chain identifiers (email addresses, social media accounts, etc.)
```

Comment:
```suggestion
This CIP proposes a new version (from here referred to as ASv2) of the existing validator-run phone number attestation protocol (CIP01, from here referred to as ASv1), which will leverage a federated set of independent issuers to perform attestations. It will also lay the foundation for new identifier types (email addresses, social media accounts, etc.) to be brought on-chain.
```